middleware/middleware-kafka/kafka-debezium/99-outro.md

```diff
@@ -1,10 +1,12 @@
-In this scenario you learned about the change data capture concept and how you can leverage Debezium for that purpose.
+In this scenario you learned about the change data capture concept and how to run Debezium to establish a CDC data flow.
 
-You have learnt what components you need to deploy a solution based on Debezium, how to deploy an Apache Kafka broker and how to deploy a Kafka Connect instance with Debezium inside and create a link between the Kafka Connect and source database.
+You learned about the components that are required to deploy a solution based on Debezium, how to deploy an Apache Kafka broker, how to deploy a Kafka Connect instance that includes Debezium, and how to create a link between Kafka Connect and source database.
 
-But this is just a beginning of a long journey. Please take your time and look at these resources:
-* Debezium [website](http://debezium.io/)
-* Extensive Debezium [tutorial](http://debezium.io/docs/tutorial/)
-* Debezium [examples repository](https://github.com/debezium/debezium-examples)
-* How to run Debezium on [Minishift](http://debezium.io/docs/openshift/)
+But this is just the beginning.
+Continue to discover more about Debezium by exploring these additional resources:
+
+* [Debezium website](http://debezium.io/)
+* [Extended Debezium tutorial](http://debezium.io/docs/tutorial/)
+* [Debezium examples repository](https://github.com/debezium/debezium-examples)
+* [How to run Debezium on Minishift](http://debezium.io/docs/openshift/)
```

Comment:
This URL goes through a Redirect. Can you add the actual one instead?

Reply:
Will do